src/index.html

```diff
@@ -1,12 +1,534 @@
 <!doctype html>
-<html lang="en">
+<html 
+  lang="en"
+  class="default">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="default__body">
+    <header class="default__header header">
+      <a 
+        href="#"
+        class="header__link">
+        <img 
+          src="./images/logo.png" 
+          alt="logo image"
+          class="header__image">
+      </a>
+      
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a 
+              href="#"
+              class="header__nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__item">
```

Comment:
Check the code style. Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
          </li>
          
          <li class="header__item">
```